app/views/catalog/_admin_controls_recording.html.erb

```diff
@@ -12,7 +12,7 @@
         <%= link_to "Order Manager", main_app.polymorphic_path([:order_manager, resource]), class: 'btn btn-default' %>
       <% end %>
     <% end %>
-    <% if can?(:update, resource) %>
+    <% if can?(:update, resource) && Wayfinder.for(resource).scanned_resources_count == 0 %>
```

Comment:
I wonder if it makes sense to check that there are files, instead of there are not scanned resources?

Reply:
I like this for consistency, and it seems reasonable to check to see if the object has any of the things that are being displayed rather than that it doesn't have something it wouldn't display. Are you concerned that one day we might have another model type as a nested resource? I have some concern that we might have both a FileSet and another type, and we'd still want to display this if so.

Reply:
Your reasoning is why I was thinking of checking for the files. if there are files, then we could create a playlist out of them, so display the playlist link. It seems like you are thinking about the member list maybe? what is the thread of consistency?